src/web/index.js

```diff
@@ -30,7 +30,7 @@ const clipboard = new Clipboard(".clipboardButton");
 const addonUrl = process.env.ADDON_URL;
 
 // Period after which app loading indicator will disappear if add-on not found
-const LOADER_DELAY_PERIOD = 500;
+const LOADER_DELAY_PERIOD = 2000;
```

Comment:
This should give us a minimum of 2 seconds for the loader, but retaining the timer extensions every time state changes during initialization